### PR TITLE
NTP-768: Update the Contact us page as per the acceptance criteria of the ticket

### PR DESCRIPTION
**Pull request by hasan3d**
## Context

Update the Contact us page as per the new content supplied.

## Changes proposed in this pull request

- Amend the 'Contact us' page with new content supplied.
- Remove blue 'contact us' banner
- Create a 'Contact us' link positioned as the first link of the footer
- Create a 'Report issues' link on the right-hand side of the footer

## Guidance to review

Ensure the new amended content is reflected on the 'Contact us' page. Also, ensure the changes proposed with this PR is satisfactory. Test with and without js enabled on mobile and check accessibility.

## Link to Jira ticket

[https://dfedigital.atlassian.net/browse/NTP-768](https://dfedigital.atlassian.net/browse/NTP-768)

## Things to check

- [x] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [x] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [x] `dotnet format` has been run in the repository root
- [x] Test coverage of new code is at least 80%
- [x] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [x] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [x] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: UI/Pages/ContactUs.cshtml [hunk 1]
```diff
@@ -9,25 +9,24 @@
...
-                    email <a href="mailto:tutoring.support@service.education.gov.uk" data-testid="mailto-ntp-link"
-                             class="govuk-link">tutoring.support@service.education.gov.uk.</a>
-                </li>
-                <li>call 0300 373 0891 - open Monday to Friday 8.30am to 5pm (except bank holidays)</li>
-            </ul>
-        </p>
-        <h2 class="govuk-heading-m">For issues with your tuition partner </h2>
-        <p class="govuk-body">
-            For help if you have an issue with your tuition partner, you can  <a asp-page="@nameof(ReportIssues)" class="govuk-footer__link" data-testid="report-issues-link">read our guidance.</a>                                                                            
+            <a data-testid="find-out-call-charges-link" class="govuk-link" target="_blank" href="https://www.gov.uk/call-charges">Find out about call charges</a>
```
Comment (ashward) [inline UI/Pages/ContactUs.cshtml:18]: We should open this in the same tab for accessibility as per https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab

```suggestion
            <a data-testid="find-out-call-charges-link" class="govuk-link" href="https://www.gov.uk/call-charges">Find out about call charges</a>
```

**Review by ashward — CHANGES_REQUESTED** [review]
Just the one about changing the call charges link to open in the same tab. Other than that it looks good!

File: UI/Pages/ContactUs.cshtml (hunk 1 above)
Comment (hasan3d) [reply UI/Pages/ContactUs.cshtml]: Okay, I have now made the suggested changes.

**Review by ashward — APPROVED** [review]
lgtm

**Pull request merged**